scanner/src/funTest/kotlin/scanners/FileCounterTest.kt

```diff
@@ -61,7 +61,7 @@ class FileCounterTest : StringSpec() {
             val expectedResult = patchExpectedResult(
                     File(assetsDir, "file-counter-expected-output-for-analyzer-result.yml"))
 
-            val ortResult = FileCounter(ScannerConfiguration()).scanDependenciesFile(analyzerResultFile, outputDir,
+            val ortResult = FileCounter(ScannerConfiguration()).scanOrtFile(analyzerResultFile, outputDir,
```

Comment:
consider using "scanOrtResultFile" as the term "OrtResult" seems to be widely used.

Reply:
Do you really mean "Ort*Result*" is widely used (like in the variable name), or "Ort*File*" is widely used (like in the function name)?

Reply:
I was referring to 

git grep -i "Ort file" | wc -l 
1

vs

git grep -i "Ort Result" | wc -l
21

Reply:
I guess the more representative check would be without a space in between:

    $ git grep -i "ortfile" | wc -l                                                                                                                                             
    22                                                                                                                                                                                                                                                                                                                                    
    $ git grep -i "ortresult" | wc -l                                                                                                                                      
    144

Anyway, I agree `scanOrtResultFile` is the most precise, but also the longest. I'll make the change.